google-cloud-storage/src/main/scala/akka/stream/alpakka/googlecloud/storage/impl/GCStorageStream.scala

```diff
@@ -106,8 +99,9 @@ import scala.util.control.NonFatal
       import mat.executionContext
       val queryParams =
         Map(
-          pageToken.map(token => "pageToken" -> token).toList ++
-          prefix.map(pref => "prefix" -> pref).toList: _*
+          pageToken.map(token => "pageToken" -> token).toList :::
+          prefix.map(pref => "prefix" -> pref).toList :::
+          List("versions" -> s"$versions").filter(_ => versions): _*
```

Comment:
What about `(if (versions) List("versions" -> true.toString) else Nil)`?

Reply:
yeah, I agree, it's a bit more readable this way